core/domain/acl_decorators.py

```diff
@@ -45,6 +45,10 @@ def test_can_play(self, exploration_id, **kwargs):
 
         Args:
             exploration_id: str. The exploration id.
+            kwargs: keyword_arguments.
```

Comment:
Here and elsewhere: `Keyword arguments.`

(It's not a python variable, so you don't need to refer to it in snake case.)

Reply:
This will require a modification of the regex since it allows only non-space characters for the argument type.

Reply:
Oh, you're using keyword_arguments as a type. It's not a type. Use `*` instead.

Reply:
Done.